### DISCUSSION TITLE
NiceGUI PDF documentation

### DISCUSSION THREAD

**Discussion by muratbedir** [zauberzeug/nicegui#3165] [Ideas / Feature Requests]
Need a pdf book of Nicegui that covers, all features with sample this book will support framework to reach more people.

**Comment by rodja** [comment]
Sure. Who likes give it a go?

**Comment by ed2050** [comment]
What's wrong with nicegui.io/docs -> print -> Save as PDF?

**Comment by ed2050** [comment]
Or perhaps you mean more of a tutorial format, rather than reference format in current docs.

I started a tutorial [here](https://github.com/ed2050/nicegui-docs/blob/main/overview.md).  The idea was to step through each feature of nicegui and build an example app with screenshots along the way.  I did the first few pages to illustrate.  The rest needs to be filled in.

Link above is just the intro page, which turned into the main nicegui docs page.  See [Contents](https://github.com/ed2050/nicegui-docs/blob/main/overview.md#contents) section for the actual tutorial pages.

**Reply by muratbedir** [reply]
Thask for starting document, 
Such a document would be very useful for beginners, but we need to develop it hand in hand. More people need to hear about and use an innovative platform like NiceGUI. I believe in the potential of NiceGUI.

**Reply by ed2050** [reply]
Agree.  The question is who will do it?  Not sure I have the time or desire without funding.

The time consuming part is developing example code and screenshots.  Once you have that, writing part is easy (at least to me).
